symphony/app/fbcnms-projects/workflows/src/routes.js

```diff
@@ -299,21 +313,23 @@ router.get('/id/:workflowId', async (req, res, next) => {
       );
     }
 
-    const subs = filter(identity)(
-      map(task => {
-        if (task.taskType === 'SUB_WORKFLOW') {
-          const subWorkflowId = task.inputData && task.inputData.subWorkflowId;
-
-          if (subWorkflowId != null) {
-            return {
-              name: task.inputData.subWorkflowName,
-              version: task.inputData.subWorkflowVersion,
-              referenceTaskName: task.referenceTaskName,
-              subWorkflowId: subWorkflowId,
-            };
+    const subs: Array<WorkflowType> = anythingTo<Array<WorkflowType>>(
+      filter(identity)(
+        map((task: WorkflowType): WorkflowType | void => {
```

Comment:
Good place for a Maybe type

Also, should we rename this type to TaskType? Since we're iterating through the tasks of a specific workflow, I think calling it WorkflowType might be misleading.

```suggestion
        map((task: WorkflowType): ?WorkflowType => {
```

Reply:
Fixed, thanks